requirements/dev-requirements.txt

```diff
@@ -11,12 +11,13 @@ sniffer==0.4.0
 psutil==5.1.3  # for memory profiling
 django-extensions==1.7.7
 ipython==5.2.2
-ipdb==0.9.4
+ipdb==0.11
```

Comment:
Should we just specify a minimum version rather than pinning to a specific one?  For dev-specific tools anyways, that seems preferable.